src/App.vue

```diff
@@ -279,12 +287,31 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
           this.saveErrorDialog = true
       }
     })
+
+    this.$root.$on('name-request-invalid-error', async error => {
+      console.log('NR error during File and Pay =', error)
```

Comment:
If this console.log message is needed then you'll have to disable the lint error.
Also below.

Reply:
I am not sure whether I understood this. I dont see any lint error locally and in PR checks

Reply:
In GH for this PR, go to Files Changed tab and look through the code. There are 2 lines with a GHA warning.

Reply:
ok